src/core/types.ts

```diff
@@ -2,21 +2,25 @@ import { FC, ComponentClass } from 'react';
 import { Store, Event, Effect } from 'effector';
 import { useUnit, useList } from 'effector-react';
 
-export interface ReflectCreatorContext {
+export interface Ctx {
```

Comment:
```suggestion
export interface Context {
```

Maybe rename it to a full name